relay-aws-extension/src/aws_extension.rs

```diff
@@ -0,0 +1,193 @@
+use actix::actors::signal::{Signal, SignalType};
+use actix::prelude::*;
+use failure::Fail;
+use relay_system::Controller;
+use reqwest::blocking::Client;
+use reqwest::StatusCode;
+use serde::Deserialize;
+use std::collections::HashMap;
+
+const EXTENSION_NAME: &str = "sentry-lambda-extension";
+const EXTENSION_NAME_HEADER: &str = "Lambda-Extension-Name";
+const EXTENSION_ID_HEADER: &str = "Lambda-Extension-Identifier";
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[allow(dead_code)]
+struct RegisterResponse {
+    pub function_name: String,
+    pub function_version: String,
+    pub handler: String,
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[allow(dead_code)]
+struct Tracing {
+    pub r#type: String,
```

Comment:
The usual way to do this is:
```suggestion
    #[serde(rename = "type")
    pub ty: String,
```